avogadro/qtplugins/cp2kinput/cp2kinputdialog.h

```diff
@@ -102,6 +102,8 @@ private slots:
 
   void setBasicDefaults();
 
+  /// @return valid values for CP2K RUN_TYPE
+  static QString toenum(CalculateOption option);
```

Comment:
Does this need to be `static`? You're only using it inside the class, right? I don't think it matters either way.

Reply:
The method does no depend on member variables but on private enum, fromCalcEnum, so it can be specified as static.